src/Raven.Server/Documents/Handlers/Debugging/MemoryDebugHandler.cs

```diff
@@ -318,6 +318,10 @@ private static void WriteMemoryStats(AsyncBlittableJsonTextWriter writer, JsonOp
                 [nameof(MemoryInfo.HighMemSinceStartup)] = memoryUsageRecords.High.SinceStartup.ToString(),
                 [nameof(MemoryInfo.LowMemSinceStartup)] = memoryUsageRecords.Low.SinceStartup.ToString(),
             };
+            if (memInfo.Remarks != null)
```

Comment:
Should add this for Linux as well (cgroup)

Reply:
Good point, yes.